tests/Integration/SearchTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Helper\Tests\Integration;
+
+use Algolia\AlgoliaSearch\Helper\SearchClient;
+
+class SearchTest extends AlgoliaIntegrationTestCase
+{
+    protected function setUp()
+    {
+        parent::setUp();
+        static::$indexes['main'] = self::safeName('indexing');
+    }
+
+    public function testIndexNotExist()
+    {
+        /** @var \Algolia\AlgoliaSearch\Helper\SearchIndex $index */
+        $index = SearchClient::get()->initIndex('IndexNotExisting');
```

Comment:
You should somehow try to have the same thing as the common test suite, create a name of the current test name.